docs/sources/flow/reference/cli/convert.md

```diff
@@ -118,3 +119,16 @@ If you have unsupported features in a Static mode source configuration, you will
 also raise warnings for configuration options that may require your attention.
 
 Refer to [Migrate from Grafana Agent Static to {{< param "PRODUCT_NAME" >}}]({{< relref "../../tasks/migrate/from-static/" >}}) for a detailed migration guide.
+
+### OpenTelemetry Collector
+
+Using the `--source-format=otelcol` will convert the source configuration from
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/configuration/)
+to {{< param "PRODUCT_NAME" >}} configuration.
+
+Many OpenTelemetry Collector components are supported. You can get a general idea of which exist in Flow mode for conversion
+by reviewing the `otelcol.*` components in the [Component Reference]({{< relref "../components/" >}}). If you have unsupported
+features in a source configuration, you will receive [errors] when you convert to a flow configuration. The converter will
+also raise warnings for configuration options that may require your attention.
+
+Refer to [Migrate from OpenTelemetry Collector to {{< param "PRODUCT_NAME" >}}]({{< relref "../../tasks/migrate/from-otelcol/" >}}) for a detailed migration guide.
```

Comment:
```suggestion
Refer to [Migrate from OpenTelemetry Collector to {{< param "PRODUCT_NAME" >}}][migrate] for a detailed migration guide.

[Component Reference]: ../../components/
[migrate]: ../../../tasks/migrate/from-otelcol/

```

We should not use relrefs in any documentation now. This shortcode has been deprecated. We can use relative URIs for links within the same doc tree which should work in both the OSS and Cloud docs.

@erikbaranowski We need to check the paths in a local build once these changes are applied. The URI sometimes has a slightly different path depth than relrefs.